internal/grpc/interceptors/auth/auth.go

```diff
@@ -280,7 +276,7 @@ func dismantleToken(ctx context.Context, tkn string, req interface{}, mgr token.
 		}
 	}
 
-	return nil, err
+	return nil, errtypes.PermissionDenied("access token has invalid scope")
```

Comment:
@ishank011 I think this is a bug and we should return an explicit permission denied error here because `extractRef` might not return `ok` ... then err is still nil.

Reply:
@butonic looks good, thanks!